util/mvmap/mvmap.go

```diff
@@ -85,6 +85,15 @@ func (ds *dataStore) get(e entry, key []byte) []byte {
 	return slice[valOffset : valOffset+e.valLen]
 }
 
+func (ds *dataStore) getEntryData(e entry) (key, value []byte) {
+	slice := ds.slices[e.addr.sliceIdx]
```

Comment:
if `sliceIdx` already target than 0, then it is safe to do so. Is that possible `sliceIdx` being negative?